functions/src/models/user.models.ts

```diff
@@ -20,7 +20,6 @@ export interface TanamUserInvited {
   email: string;
   invited: any;
   role: UserRole;
-  uid: string;
```

Comment:
I think that we might still need the `uid` no? But it can be null, so that we can use the interface as a "pending invitation".

```typescript
uid?: string;
```

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/257/commits/0822192a47e16d5da4de432c77f9bc6ef519a0a3